gce-staging-1/k8s.tf

```diff
@@ -0,0 +1,46 @@
+resource "random_id" "gke_username" {
+  byte_length = 32
+}
+
+resource "random_id" "gke_password" {
+  byte_length = 32
+}
+
+resource "google_container_cluster" "gke-staging-1" {
+  name               = "gke-staging-1"
+  zone               = "us-central1-a"
+  initial_node_count = 3
+
```

Comment:
What types of instances are the nodes in this cluster? Might be good to make it explicit, as I'm not sure what the default is.

My personal experience is that the nodes should probably be ~4GB RAM VMs. I'm running a personal cluster with 2GB and it is awfully tight on memory, even without much running on it.